examples/component-versioning/component-mutable.md

```diff
@@ -2,9 +2,11 @@
 
 ## Prerequisite
 
-1. Make sure [`addon-oam-kubernetes-local`](https://github.com/crossplane/addon-oam-kubernetes-local) was installed.
-2. Before component versioning mechanism released in Crossplane, you can run `go run examples/containerized-workload/main.go`
-   instead of using crossplane. After this feature was released in crossplane, install crossplane will work.
+   Start OAM runtime:
+
+   ```shell script
+    go run cmd/oam-kubernetes-runtime/main.go
+   ```
```

Comment:
Ref to in [the installation step](https://github.com/crossplane/oam-kubernetes-runtime#install-oam-runtime) insteand of `go run`.